packages/ncids-css/packages/usa-button/internal/_external.scss

```diff
@@ -1,24 +1,19 @@
-@import '../../../core/mixins/external_check';
+@use 'uswds-core' as *;
```

Comment:
if we're not using this file, let's get rid of it. we can keep all the changes you need in a wiki or something else for now